spec/features/merchants/discounts/index_spec.rb

```diff
@@ -38,19 +38,46 @@
 
   it "will only display a merchants discounts" do
     within "#discount-#{@discount_1.id}" do
-      expect(page).to have_link("#{@discount_1.name}")
-      expect(page).to have_content("Percent Off: #{@discount_1.percent_off}")
+      expect(page).to have_link("#{@discount_1.id}")
```

Comment:
Changed the name links to no longer be links, instead the discount id's are the links. Remove edit link on the index page